sfdx-source/apex-common/test/classes/fflib_SObjectDomainTest.cls

```diff
@@ -172,10 +172,9 @@ private with sharing class fflib_SObjectDomainTest
 		Opportunity opp = new Opportunity ( Name = 'Test Recursive 1', Type = 'Existing Account' );
 		fflib_SObjectDomain.Test.Database.onInsert(new Opportunity[] { opp } );		
 		fflib_SObjectDomain.triggerHandler(fflib_SObjectDomain.TestSObjectStatefulDomainConstructor.class);		
-		System.assertEquals(2, fflib_SObjectDomain.Errors.getAll().size());		
+		System.assertEquals(1, fflib_SObjectDomain.Errors.getAll().size());
```

Comment:
since this behavior is a bug we should rather comment the test out and make the rest part of the contribution

Reply:
Sounds good